pyproject.toml

```diff
@@ -1,3 +1,6 @@
 [build-system]
 requires = ["setuptools"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[tool.black]
```

Comment:
if you insist on using black (instead of flake8), please make sure to have the same configuration as flake8.

Reply:
No - I don't have any strong preferences - it is just very helpful for me to have one formatter defined. Removed it & switched to autopep8 :)